source/common/http/filter/gzip_filter.cc

```diff
@@ -36,7 +44,7 @@ FilterHeadersStatus GzipFilter::encodeHeaders(HeaderMap& headers, bool end_strea
   }
 
   return Http::FilterHeadersStatus::Continue;
-} // namespace Http
+}
```

Comment:
Please keep this namespace comment in.

Reply:
It looks like that ```namespace Http``` was in a wrong place. It should be in the line 128 only.